thirdparty_build.gradle

```diff
@@ -9,8 +9,25 @@ buildscript {
 /*DJVU CHAIN*/
 task downloadDjvu {
     doLast {
-        exec {
-            commandLine = ['git', 'clone', 'git://git.code.sf.net/p/djvu/djvulibre-git', djvu]
+        def djvuFolder = new File(djvu)
+        if (djvuFolder.exists()) {
+            // Clean previous checkout
+            exec {
+                commandLine = ['git', '-C', djvu, 'checkout', '.']
```

Comment:
This is quite unusual approach for me to clean previous checkout. (I wonder what is origin of this approach?)
Why don't use just clean or reset?

Looks like that first two steps could be combined, couldn't it?

Reply:
We are working in the detached head here and just pulling will result in the error, I used checkout to move to HEAD. I have fixed this hack.